lib/grpc/connection.js

```diff
@@ -255,6 +255,9 @@ class GrpcConnection extends EventEmitter {
         // per the spec, An UNIMPLEMENTED status code from gRPC indicates
         // that the versioned Trace Observer is no longer available. Agents
         // MUST NOT attempt to reconnect in this case
+        logger.info(
+          '[UNIMPLEMENTED]: Trace Obserserver is no longer available. Shutting down connection.'
+        )
```

Comment:
This could be inferred before but being explicit may help speed up troubleshooting.